[Bug] [datavines.core] When I log in to the system using admin, an error message is displayed 和 the login page is displayed.

### Search before asking

- [X] I had searched in the [issues](https://github.com/datavines-ops/datavines/issues?q=is%3Aissue+label%3A%22bug%22) and found no similar issues.


### What happened

When I log in to the system using admin, an error message is displayed and the login page is displayed。
![20240715_155535](https://github.com/user-attachments/assets/19529e23-d91e-46df-8d73-ec7a061d2e59)


### DataVines Version

dev

### DataVines Config

```conf
mysql
```


### Running Command

```shell
sh datavines-daemon.sh start mysql
```


### Error Exception

```log
[ERROR] 2024-07-15 15:44:32.593 io.datavines.server.api.inteceptor.DataVinesExceptionHandler:[50] - DataVinesServerException:
io.datavines.core.exception.DataVinesServerException: Token is Null Error
```


### Engine Type

_No response_

### Java Version

jdk1.8

### Screenshots

_No response_

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

--- a/datavines-core/src/main/java/io/datavines/core/entity/ResultMap.java
+++ b/datavines-core/src/main/java/io/datavines/core/entity/ResultMap.java
@@ -69,11 +69,6 @@ public class ResultMap extends HashMap<String, Object> {
                     throw new DataVinesServerException(Status.TOKEN_IS_NULL_ERROR);
                 }
             }
-        } else {
-            token = request.getParameter(DataVinesConstants.TOKEN_HEADER_STRING);
-            if (StringUtils.isEmpty(token)) {
-                throw new DataVinesServerException(Status.TOKEN_IS_NULL_ERROR);
-            }
         }
         this.code = Status.SUCCESS.getCode();
         this.put("code", this.code);
@@ -110,11 +105,6 @@ public class ResultMap extends HashMap<String, Object> {
                 if (StringUtils.isEmpty(token)) {
                     throw new DataVinesServerException(Status.TOKEN_IS_NULL_ERROR);
                 }
-            }
-        } else {
-            token = request.getParameter(DataVinesConstants.TOKEN_HEADER_STRING);
-            if (StringUtils.isEmpty(token)) {
-                throw new DataVinesServerException(Status.TOKEN_IS_NULL_ERROR);
             }
         }
 

--- a/datavines-server/src/main/java/io/datavines/server/dqc/coordinator/quartz/CatalogTaskScheduleJob.java
+++ b/datavines-server/src/main/java/io/datavines/server/dqc/coordinator/quartz/CatalogTaskScheduleJob.java
@@ -31,12 +31,12 @@ import org.slf4j.LoggerFactory;
 
 import java.time.LocalDateTime;
 
-public class CommonTaskScheduleJob implements org.quartz.Job {
+public class CatalogTaskScheduleJob implements org.quartz.Job {
 
     /**
      * logger of FlowScheduleJob
      */
-    private static final Logger logger = LoggerFactory.getLogger(CommonTaskScheduleJob.class);
+    private static final Logger logger = LoggerFactory.getLogger(CatalogTaskScheduleJob.class);
 
     public JobExternalService getJobExternalService(){
         return SpringApplicationContext.getBean(JobExternalService.class);

--- a/datavines-server/src/main/java/io/datavines/server/repository/service/impl/CommonTaskScheduleServiceImpl.java
+++ b/datavines-server/src/main/java/io/datavines/server/repository/service/impl/CommonTaskScheduleServiceImpl.java
@@ -24,7 +24,7 @@ import io.datavines.core.enums.Status;
 import io.datavines.core.exception.DataVinesServerException;
 import io.datavines.server.api.dto.bo.task.CommonTaskScheduleCreateOrUpdate;
 import io.datavines.server.api.dto.bo.job.schedule.MapParam;
-import io.datavines.server.dqc.coordinator.quartz.CommonTaskScheduleJob;
+import io.datavines.server.dqc.coordinator.quartz.CatalogTaskScheduleJob;
 import io.datavines.server.dqc.coordinator.quartz.QuartzExecutors;
 import io.datavines.server.dqc.coordinator.quartz.ScheduleJobInfo;
 import io.datavines.server.dqc.coordinator.quartz.cron.StrategyFactory;
@@ -110,7 +110,7 @@ public class CommonTaskScheduleServiceImpl extends ServiceImpl<CommonTaskSchedul
         switch (JobScheduleType.of(scheduleCreateOrUpdate.getType())) {
             case CYCLE:
             case CRONTAB:
-                quartzExecutor.addJob(CommonTaskScheduleJob.class, getScheduleJobInfo(commonTaskSchedule));
+                quartzExecutor.addJob(CatalogTaskScheduleJob.class, getScheduleJobInfo(commonTaskSchedule));
                 break;
             case OFFLINE:
                 break;

